stable/prometheus-operator/hack/sync_prometheus_rules.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python3
+"""Fetch rules from charts dictionary into this chart."""
+import textwrap
+
+import requests
+import yaml
+from yaml.representer import SafeRepresenter
+
+
+# https://stackoverflow.com/a/20863889/961092
+class LiteralStr(str):
+    pass
+
+
+def change_style(style, representer):
+    def new_representer(dumper, data):
+        scalar = representer(dumper, data)
+        scalar.style = style
+        return scalar
+
+    return new_representer
+
+
+# Source files list
+charts = [
+    {
+        'source': 'https://raw.githubusercontent.com/coreos/prometheus-operator/master/contrib/kube-prometheus/manifests/prometheus-rules.yaml',
```

Comment:
Note that eventually we will stop checking in the generated code and an artifact will only be available on kube-prometheus releases. That doesn't prevent this strategy, just means updating is likely going to be possible less frequently.

Reply:
Thanks for review! Sounds good enough for me. Maybe we should to add a note about running rules sync script on appVersion bump, and maybe add check to CI run - I'll see what I can do.